source/includes/gcp/_backend_services.md

```diff
@@ -1 +1,98 @@
+      "connectionDraining": {
+        "drainingTimeoutSec": 300
+      },
+      "creationTimestamp": "2019-09-23T12:57:58.039-07:00",
+      "description": "",
+      "enableCDN": false,
+      "fingerprint": "nH5X-L6Fovs=",
+      "healthChecks": [
+        "https://www.googleapis.com/compute/v1/projects/testAreaName/global/healthChecks/healthCheckName"
+      ],
+      "kind": "compute#backendService",
+      "loadBalancingScheme": "EXTERNAL",
+      "port": 80,
+      "portName": "http",
+      "protocol": "HTTP",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/testAreaName/global/backendServices/backendServiceName",
+      "sessionAffinity": "NONE",
+      "timeoutSec": 30,
+      "shortHealthChecks": "healthCheckName",
+      "shortBackends": "instanceGroupName",
+      "id": "526709884866305321",
+      "name": "backendServiceName"
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/backendservices</code>
+
+Retrieve a list of all backend services in an [environment](#administration-environments).
+
+| Attributes                          | &nbsp; |
+| ----------------------------------- | ------ |
+| `affinityCookieTtlSec`<br/>*number* | If set to 0, the cookie is non-persistent and lasts only until the end of the browser session (or equivalent). The maximum allowed value is one day (86,400).|
+| `backends`<br/>*object*             | The list of backends that serve this BackendService.|
+| `connectionDraining`<br/>*object*   | - |
+| `creationTimestamp`<br/>*string*    | Creation timestamp in RFC3339 text format. |
+| `description`<br/>*string*          | An optional description of this resource. Provide this property when you create the resource.|
+| `enableCDN`<br/>*boolean*           | If true, enables Cloud CDN for the backend service. Only applicable if the `loadBalancingScheme` is EXTERNAL and the protocol is HTTP or HTTPS. |
+| `fingerprint`<br/>*string*          | Fingerprint of this resource. A hash of the contents stored in this object. |
+| `healthChecks`<br/>*string*         | The list of URLs to the HttpHealthCheck or HttpsHealthCheck resource for health checking this BackendService. Currently at most one health check can be specified. |
+| `id`<br/>*string*                   | The unique identifier for the resource. This identifier is defined by the server. |
+| `kind`<br/>*string*                 | Type of resource. Always compute#backendService for backend services.|
+| `loadBalancingScheme`<br/>*enum*    | Indicates whether the backend service will be used with internal or external load balancing. Possible values are INTERNAL and EXTERNAL.|
+| `name`<br/>*string*                 | Name of the resource. |
+| `port`<br/>*number*                 | Deprecated in favor of portName. The TCP port to connect on the backend. The default value is 80. |
```

Comment:
I would just put: The TCP port to connect on the backend. The default value is 80
Or remove it, I am not sure we want to document deprecated stuff, even if google does (well they have to it's their API)

Reply:
I removed it.